src/policies/InvalidationPolicyManager.ts

```diff
@@ -101,9 +101,19 @@ export default class InvalidationPolicyManager {
     if (!typePolicyForEvent) {
       return;
     }
-    Object.keys(typePolicyForEvent).forEach((typeName: string) => {
-      const typeMapEntities = entityTypeMap.readEntitiesByType(typeName) ?? {};
-      const policyAction = typePolicyForEvent[typeName];
+
+    const { __default: defaultPolicyAction, ...restTypePolicyTypeNames } = typePolicyForEvent;
+
+    if (defaultPolicyAction) {
+      defaultPolicyAction(mutedCacheOperations, {
+        storage: this.getPolicyActionStorage(`${typeName}__default`),
+        ...policyMeta,
+      });
```

Comment:
@danReynolds Not sure I follow exactly what this does function call accomplishes. `mutedCacheOperations` is an instance variable, is that correct?